app/src/main/java/com/kickstarter/viewmodels/ProjectCardHolderViewModel.kt

```diff
@@ -307,27 +326,25 @@ interface ProjectCardHolderViewModel {
                 .subscribe(this.heartDrawableId)
```

Comment:
I believe we have to use a lambda expression here: 
`.subscribe { this.heartDrawableId.onNext(it)`

Reply:
good catch!